src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx

```diff
@@ -127,7 +127,7 @@
     <value>Please specify one tool Package Id to uninstall.</value>
   </data>
   <data name="CommandDescription" xml:space="preserve">
-    <value>Uninstall a tool from the current development environment.</value>
+    <value>Uninstall a global tool.</value>
```

Comment:
This one does not affect local tools as well?

Reply:
agree, we need to add local tools here

Reply:
Added local tools